.github/workflows/reusable-semgrep-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: Semgrep
+
+on:
+  workflow_call:
+    inputs:
+      github_event_name:
+        required: true
+        type: string
+      github_pr_sha:
```

Comment:
The name is not clear. Which sha is it? head sha, base sha, something else?

Reply:
We can actually inherit the context here, removing them